spec/controllers/tokens_controller_spec.rb

```diff
@@ -41,16 +43,15 @@
     end
   end
 
-  describe 'authorize response memoization' do
-    it "memoizes the result of the authorization" do
+  describe 'authorize response memorization' do
+    it 'memorizes the result of the authorization' do
+      skip 'No assertion.'
```

Comment:
These `skip`s are super useful, these specs are wrong and it's good to mark them as such. Thanks!! :+1: